enable/trait_defs/ui/kiva_font_editor.py

```diff
@@ -23,6 +23,20 @@
 from .editor_with_component import EditorWithLabelComponent
 
 
+WEIGHTS = {
```

Comment:
There are several unused imports in this file; not introduced in this PR, I know, but they may be worth cleaning up in passing.